src/libraries/Microsoft.Extensions.FileSystemGlobbing/ref/Microsoft.Extensions.FileSystemGlobbing.cs

```diff
@@ -195,18 +195,18 @@ public partial struct FrameData
             public bool InStem;
```

Comment:
`PatternContextLinear`'s ctor doesn't verify that `pattern` is not `null`. Should it check and throw ArgNull?